src/rust/engine/process_execution/src/remote.rs

```diff
@@ -372,8 +372,9 @@ impl super::CommandRunner for CommandRunner {
 
                             // take the grpc result and cancel the op if too much time has passed.
                             let elapsed = start_time.elapsed();
+                            let queue_buffer_time = std::time::Duration::from_secs(45);
```

Comment:
I went back and forth between 30, 45, and 60. Went with 45 based on the fact that a test that takes 3.5 seconds for actual the actual Pytest process timed out after 60 seconds. That implies to me that 55 seconds were for queuing. Went with 45 as a middle ground.